### DISCUSSION TITLE
What's coming in version 3?

### DISCUSSION THREAD

**Discussion by darwayne** [pion/webrtc#1419] [General] [answered]
Hey there!

Huge pion fan here; I'm currently using version 2 I see version 3 is getting worked on what can we expect to see in version 3? 

Thanks!

**Comment by Sean-Der** [comment, marked as answer]
Hey @darwayne, thanks for the kind words :)

I have started https://github.com/pion/webrtc/wiki/Release-WebRTC@v3.0.0 but we have some other stuff in https://github.com/pion/webrtc/projects/18 I haven't documented yet

I will be filling this out soon. I have been more focused on https://webrtcforthecurious.com recently.

**Reply by darwayne** [reply]
Oooh very nice. Thanks @Sean-Der
